pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -34,8 +36,12 @@ func newMemContinueHint(continueTarget hinter.ResOperander, memset bool) hinter.
 				return err
 			}
 
-			newN := new(f.Element)
-			newN = newN.Sub(n.(*f.Element), &utils.FeltOne)
+			newN, ok := n.(fp.Element)
+			if !ok {
+				return fmt.Errorf("casting n into a felt failed")
+			}
+
+			newN.Sub(&newN, &utils.FeltOne)
 
 			if err := ctx.ScopeManager.AssignVariable("n", newN); err != nil {
```

Comment:
Can you change this to `ctx.ScopeManager.AssignVariable("n", *newN)` too?

Reply:
its not needed
newN is now a fp.element, not a *fp.element

Reply:
ah lol didn't notice the assignment wasn't happening anymore